README.md

```diff
@@ -26,53 +26,73 @@ If you create a PostgreSQL data source using the data source generator as descri
 
 ## Creating a data source
 
-Use the [Data source generator](http://loopback.io/doc/en/lb3/Data-source-generator.html) to add a PostgreSQL data source to your application.  
-The generator will prompt for the database server hostname, port, and other settings
-required to connect to a PostgreSQL database. It will also run the `npm install` command above for you.
-
-The entry in the application's `/server/datasources.json` will look like this:
-
-{% include code-caption.html content="/server/datasources.json" %}
-
-```javascript
-"mydb": {
-  "name": "mydb",
-  "connector": "postgresql"
-  "host": "mydbhost",
-  "port": 5432,
-  "url": "postgres://admin:admin@mydbhost:5432/db1?ssl=false",
-  "database": "db1",
-  "password": "admin",
-  "user": "admin",
-  "ssl": false
-}
+For LoopBack 4 users, use the LoopBack 4 [Command-line interface](https://loopback.io/doc/en/lb4/Command-line-interface.html) to generate a DataSource with PostgreSQL connector to your LB4 application. Run [`lb4 datasource`](https://loopback.io/doc/en/lb4/DataSource-generator.html), it will prompt for configurations such as host, post, etc. that are required to connect to a PostgreSQL database.
+
+After setting it up, the configuration can be found under `src/datasources/<DataSourceName>.datasource.ts`, which would look like this:
+
+```ts
+const config = {
+  name: 'db',
+  connector: 'postgresql',
+  url: '',
+  host:'localhost',
+  port: 5432,
+  user: 'user',
+  password: 'pass',
+  database: 'testdb',
+};
 ```
 
-Edit `datasources.json` to add other properties that enable you to connect the data source to a PostgreSQL database.
+<details>
+  <summary><strong>For LoopBack 3 users</strong></summary>
+  <p>Use the <a href="http://loopback.io/doc/en/lb3/Data-source-generator.html">Data source generator</a> to add a PostgreSQL data source to your application.<br />
+The generator will prompt for the database server hostname, port, and other settings
+required to connect to a PostgreSQL database. It will also run the <code class="language-plaintext highlighter-rouge">npm install</code> command above for you.</p>
+
+<p>The entry in the application’s <code class="language-plaintext highlighter-rouge">/server/datasources.json</code> will look like this:</p>
+
+<div id="/server/datasources.json" class="code-caption">
+  <p>/server/datasources.json</p>
+</div>
+
+<div class="language-javascript highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="dl">"</span><span class="s2">mydb</span><span class="dl">"</span><span class="p">:</span> <span class="p">{</span>
+  <span class="dl">"</span><span class="s2">name</span><span class="dl">"</span><span class="p">:</span> <span class="dl">"</span><span class="s2">mydb</span><span class="dl">"</span><span class="p">,</span>
```

Comment:
is it possible to wrap the json snippet into one block? Wrapping it every line makes the MD file hard to read and maintain.

Reply:
I'm afraid that that doesn't work. I tried to just wrap it with `<code>` but the format just doesn't right :p 
If we are okay with the collapsible section we should find a way to allow markdown inside of it.

Reply:
Personally, I'm good with the collapsible section. It's just the current form makes it very hard to maintain in the future. :)

Reply:
This can be fixed by having the tags with the following format:
```
<details><summary markdown="span"><strong>For LoopBack 3 users</strong></summary>
...content
</details>
``` 
and setting `parse_block_html: true` in https://github.com/strongloop/loopback.io/pull/990